lib/ReactViews/Notification/shareConvertNotification.tsx

```diff
@@ -0,0 +1,120 @@
+import i18next from "i18next";
+import { runInAction } from "mobx";
+import React from "react";
+import isDefined from "../../Core/isDefined";
+import ViewState from "../../ReactViewModels/ViewState";
+import Collapsible from "../Custom/Collapsible/Collapsible";
+import parseCustomMarkdownToReact from "../Custom/parseCustomMarkdownToReact";
+
+const Text: any = require("../../Styled/Text").default;
+const Spacing: any = require("../../Styled/Spacing").Spacing;
+const RawButton: any = require("../../Styled/Button").RawButton;
+const TextSpan: any = require("../../Styled/Text").TextSpan;
+
+export const shareConvertNotification = (shareProps: any) => (
+  viewState: ViewState
+) => {
+  const messagesForPath: { [path: string]: string[] } = {};
+  shareProps.messages?.forEach((message: any) => {
+    let pathString = message.path?.join(": ");
+    if (!pathString || pathString === null || pathString === "")
+      pathString = "root";
+    isDefined(messagesForPath[pathString])
+      ? messagesForPath[pathString].push(message.message)
+      : (messagesForPath[pathString] = [message.message]);
+  });
+
+  const rootMessages = messagesForPath["root"];
+  delete messagesForPath["root"];
+
+  const showHelp = () => {
+    viewState.showHelpPanel();
+    viewState.selectHelpMenuItem("storymigration");
```

Comment:
Also where did you end up getting the key? I was surprised cause from my view it just looks like you guessed it, but of course not

Reply:
Oh, I put it in the ticket, of course.

Reply:
stole it from https://github.com/TerriaJS/terriajs/issues/4796#issuecomment-700572601